client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -0,0 +1,253 @@
+import { Gridicon } from '@automattic/components';
+import { hasCheckoutVersion } from '@automattic/wpcom-checkout';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect, useCallback, useState } from 'react';
+import type { AkismetProQuantityDropDownProps } from './types';
+import type { FunctionComponent } from 'react';
+
+interface CurrentOptionProps {
+	open: boolean;
+}
+
+const AkismetSitesSelect = styled.div`
+	margin-top: 28px;
+`;
+
+const AkismetSitesSelectHeading = styled.div`
+	font-size: ${ hasCheckoutVersion( '2' ) ? '14px' : 'inherit' };
```

Comment:
The `${ hasCheckoutVersion( '2' ) }` code is actually only temporarily added for our recent Checkout Redesign project, since this doesn't include Akismet or Jetpack checkout it may be good to remove these ternaries. @sirbrillig I'm not sure this would harm the end user experience at all, it may just be something we have to remember to clean up later if this PR is merged with the function in place.

Reply:
I've removed the `hasCheckoutVersion` ternaries (and import). 👍
Thanks so much for reviewing and for your feedback @jjchrisdiehl! 🙌